internal/services/manager.go

```diff
@@ -21,8 +23,12 @@ type Manager struct {
 	brokerManager *brokers.Manager
 	pamService    pam.Service
 	nssService    nss.Service
+
+	quitGrpCleanup chan struct{}
 }
 
+var grpCleanInterval = 24 * time.Hour
```

Comment:
`const`?

Reply:
We need to override this value for the tests, so it can't be a const.